packages/@lwc/module-resolver/src/__tests__/mapping.spec.ts

```diff
@@ -5,6 +5,7 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
+import '../../scripts/jest/types';
```

Comment:
Instead of importing the types, the overrides should be added as ambient types. You can find a concrete example of this in the vue-next repo: 
- https://github.com/vuejs/vue-next/blob/master/packages/global.d.ts#L19-L25
- https://github.com/vuejs/vue-next/blob/master/tsconfig.json#L28

Reply:
I'm not sure this works in the case of the `module-resolver` matchers. Those come from Jest setup scripts, but adding the ambient types into those setup scripts doesn't seem to be recognized by TypeScript in the actual tests. Hence the explicit import.